### ISSUE TITLE
check the newly added maps for issues

### ISSUE THREAD

**Issue by sevu** [wesnoth/wesnoth#2453]
There was an issue in one of the maps added in fafce20, I guess it would be good to check the others too. I leave this here as issue, currently I have not the time for it.

- [ ] 2p_Clearing_Gushes
- [ ] 2p_Hellhole
- [ ] 2p_Ruined_Passage
- [ ] 2p_Ruphus_Isle
- [ ] 2p_Swamp_of_Dread
- [ ] 2p_The_Walls_of_Pyrennis
- [x] 2p_Tombs_of_Kesorak
- [ ] 4p_Bath_of_Glory
- [ ] 4p_Geothermal

**Comment by sevu** [comment]
@ln-zookeeper 
The use of the [terrain_graphics] tag in 2p_The_Walls_of_Pyrennis looks strange to me. Is that alright? (And it's the only place in multiplayer which uses [terrain_graphics].

**Comment by CelticMinstrel** [comment]
I think it's probably okay, but I also think it could likely be replaced by a now-core overlay terrain?

**Comment by Vultraz** [comment]
I don't think it can, since those hexes are meant to be impassable, and you can't give a terrain two overlays.

**Comment by CelticMinstrel** [comment]
True, true. (I didn't look closely at the map.)

**Comment by sevu** [comment]
The reason I think this could be a problem is that for joining MP games which use custom terrain I have to have the add-on installed. IIRC it's already logging errors in stderr because creating the map preview doesn't work then.
I wonder if it is a problem if I try to join a game which uses this map, but do not have it installed myself.
Admittedly, probably all players will have it since I remember no packaging constellation which splits multiplayer apart.

**Comment by CelticMinstrel** [comment]
I also don't think that's a problem when the `[terrain_graphics]` tag is placed inside the scenario - isn't scenario tag basically what's sent across the network if the other side doesn't have the addon? So as long as it doesn't reference any non-core images there should be no issue.

But yeah, this is in the core multiplayer, so I doubt there'd be a situation where one side has it and the other doesn't.
